src/helpers/role-label-formatter.js

```diff
@@ -0,0 +1,24 @@
+const RIGHT_NAME_WORDS_SEPARATOR = "_";
+const RIGHT_NAME_OMITTED_WORDS = ["gql", "mutation", "perms"];
+const QUERY_STRING = "query";
+const SEARCH_STRING = "search";
+const WHITESPACE = " ";
+const WORD_REGEX = /^(\w|\s\w)/g;
+
+const capitalizeFirstLetter = (string) => {
+  return string.toLowerCase().replace(WORD_REGEX, (letter) => letter.toUpperCase());
```

Comment:
This could be done without regex

```
const capitalizeFirstLetter = (string) => {
  return string.split(' ').map(word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()).join(' ');
};
```

Reply:
Good idea, thanks!